src/org/alicebot/ab/Category.java

```diff
@@ -295,7 +295,7 @@ public static String categoryToAIML(Category category) {
       String[] splitPattern = pattern.split(" ");
       String rpattern = "";
       for (String w : splitPattern) {
-        if (w.startsWith("<SET>") || w.startsWith("<BOT") || w.startsWith("NAME=")) {
+        if (w.startsWith("<SET>") || w.startsWith("</SET>") || w.startsWith("<BOT") || w.startsWith("</BOT>") || w.startsWith("NAME=")) {

```

Comment:
should we be case insensitive here?  we made changes a while back to be insensitive to case for the xml tags...

Reply:
i'm not positive, but its about "generating" xml - so its prolly important to pick a case and stick to it.